### ISSUE TITLE
Webauthenticator Apple Sign in

### ISSUE THREAD

**Issue by planyazilim** [dotnet/maui#16602]
### Description

Is Webauthenticator Apple Sign in working? if yes documentation is not clear, i m struggling for that for 3 days. I have tried so many things, 

If no what's your suggestion for Apple Sign in for Maui App.

1. In Startup.cs

            .AddApple(a =>
            {
                a.ClientId = "myClientId";
               a.KeyId = "muKeyId";
               a.TeamId = "myTeamID";

                var withPem =string.Format("-----BEGIN PRIVATE KEY-----\n{0}\n-----END PRIVATE KEY-----", "myp8file.p8");

                a.PrivateKey = (keyId, _) =>
                {
                    return Task.FromResult(withPem.AsMemory());
                };

                a.SaveTokens = true;
                a.GenerateClientSecret = true;
            })
2. my Back-End project is on Azure. I gave the permission WEBSITE_LOAD_USER_PROFILE = 1
3. ReturnUrl' is https://myurl/signin-apple
4. Apple User And Passwrod Screen comes, after filling them correcty Error returns
5. An unhandled exception occurred while processing the request.
CryptographicException: The system cannot find the file specified.
System.Security.Cryptography.CngKey.Import(ReadOnlySpan<byte> keyBlob, string curveName, CngKeyBlobFormat format, CngProvider provider)

Exception: An error was encountered while handling the remote login.
Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler<TOptions>.HandleRequestAsync()

Regards

### Version with bug

7.0.49

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android

### Did you find any workaround?

no

### Relevant log output

```shell
no
```

**Comment by planyazilim** [comment]
I have found the solution. My back end project is on Azure. 
WEBSITE_LOAD_USER_PROFILE should be set to 1.
I did it but i didnt know it was not working on Free Web App Service. Minimum Basic Tier is needed.
Now I have problem,name doesnt returns back from WebAuthenticator. Email is problematic, sometimes returns sometimes dont. And I didnt see any documentation about this. I found 2020 issue closed but it seems no solution.
Regards
